server/ssl/ssl.go

```diff
@@ -8,13 +8,18 @@ import (
 
 // from https://medium.com/@kelseyhightower/optimizing-docker-images-for-static-binaries-b5696e26eb07
 
-var pool *x509.CertPool
```

Comment:
Using a package scoped variable without concurrency protection is a bad idea. Switching to dependency injection only instead.